src/border_agent/border_agent.cpp

```diff
@@ -163,6 +164,8 @@ BorderAgent::BorderAgent(otbr::Ncp::RcpHost &aHost, Mdns::Publisher &aPublisher)
     , mBaseServiceInstanceName(OTBR_MESHCOP_SERVICE_INSTANCE_NAME)
 {
     mHost.AddThreadStateChangedCallback([this](otChangedFlags aFlags) { HandleThreadStateChanged(aFlags); });
+    mIsEphemeralKeyEnabled = (otThreadGetVersion() >= OT_THREAD_VERrSION_1_4);
```

Comment:
better to put this in initialization list

Reply:
accept, thanks!